recipes/mosquitto/mosquitto_1.4.10.oe

```diff
@@ -0,0 +1,3 @@
+require ${PN}.inc
+LIBRARY_VERSION = "1"
+SRC_URI += "file://0001-dont_strip.patch"
```

Comment:
Exactly what I say in the commit message. If we let the Makefile's install target do the stripping, there are no debug symbols for oe-lite's do_install postfunc do_install_strip to find, making the -dbg packages empty and useless.